packages/core/src/amazonqFeatureDev/client/featureDev.ts

```diff
@@ -26,8 +26,17 @@ import { createCodeWhispererChatStreamingClient } from '../../shared/clients/cod
 import { getClientId, getOptOutPreference, getOperatingSystem } from '../../shared/telemetry/util'
 import { extensionVersion } from '../../shared/vscode/env'
 
+// Re-enable once BE is able to handle retries.
+const writeAPIRetryOptions = {
+    maxRetries: 0,
```

Comment:
this is still at zero, so what effect does this PR have?

Reply:
It ensure Read APIs using default 3 retries. I don't handle write retries at this moment, but we would support it in the future.